src/arwave.py

```diff
@@ -1,122 +1,214 @@
 import numpy as np
-import pylab as plt
+from scipy import ndimage
+from scipy import interpolate
+
 from linetools.spectra import xspectrum1d
 from astropy.io import fits
 from astropy import units as u
 
+import ararc as aarc
```

Comment:
For consistency with other parts of the code, it might be better to just use:

`import ararc`

without changing the namespace.